language-guides/_includes/driver-warning.adoc

```diff
@@ -0,0 +1,10 @@
+
+[NOTE]
+====
+From Neo4j version 4.0 and onwards, the default encryption setting is *off* by default and Neo4j will no longer generate self-signed certificates when none is provided.
```

Comment:
Make version 4.0  bold

Reply:
Mention that the docker image, default installations and desktop come without certificates.
Aura has a cert afaik.